draft-ietf-tls-dtls13.md

```diff
@@ -1593,12 +1593,23 @@ sent in the same UDP datagram as handshake records.
 
 record_numbers:
 : a list of the records containing handshake messages in the current
-  flight which the endpoint has received, in numerically increasing
-  order. ACKs only cover the current outstanding flight (this is
-  possible because DTLS is generally a lockstep protocol). Thus, an ACK
-  from the server would not cover both the ClientHello and the client's
-  Certificate. Implementations can accomplish this by clearing their ACK
-  list upon receiving the start of the next flight.
+  flight which the endpoint has received and either processed or buffered,
+  in numerically increasing
+  order.
+
+Implementations MUST NOT acknowledge records containing
+non-duplicative handshake messages or fragments which have not been
+processed or buffered. Otherwise, deadlock can ensue.
+
+During the handshake. ACKs only cover the current outstanding flight (this is
+possible because DTLS is generally a lockstep protocol). Thus, an ACK
+from the server would not cover both the ClientHello and the client's
+Certificate. Implementations can accomplish this by clearing their ACK
```

Comment:
```suggestion
Certificate. Implementations can address this by clearing their ACK
```

Reply:
Why do you say "address" rather than accomplished? I don't necessarily object, but...